resources/update-ljm.sh

```diff
@@ -2,15 +2,49 @@
 
 set -e -u
 
+if [[ ! -z $(git status -s) ]]; then
+    echo "Git tree is not clean, aborting!"
+    exit 1
+fi
+
+BRANCH=$(git rev-parse --abbrev-ref HEAD)
+if [[ "$BRANCH" != "master" ]]; then
+  echo "Not on master, aborting!";
+  exit 1;
+fi
+
 THIS_DIR=$(cd -P "$(dirname "$(readlink "${BASH_SOURCE[0]}" || echo "${BASH_SOURCE[0]}")")" && pwd)
-LATEST_LJM_COMMIT=$(git ls-remote https://github.com/jitsi/lib-jitsi-meet.git HEAD | awk '{ print $1 }')
+PID=$$
+LJM_TMP="${TMPDIR:-/tmp}/ljm-${PID}"
 
 pushd ${THIS_DIR}/..
+CURRENT_LJM_COMMIT=$(jq -r '.dependencies."lib-jitsi-meet"' package.json | cut -d "#" -f2)
+popd
+
+git clone --branch master --single-branch --bare https://github.com/jitsi/lib-jitsi-meet ${LJM_TMP}
```

Comment:
That's rather expensive. Can we assume it's checked out in `../lib-jitsi-meet`, do a `git fetch` and read the commit through the `origin/master` branch?

Reply:
Not a blocking comment, just a nit

Reply:
It's not that expensive because we don't get the whole history, just the master branch and don't checkout the files (--bare) so in general it's very fast.

I thought of assuming that, but I didn't want to potentially screw whatever WIP stuff in here...

Reply:
Sure. It's an 11MB download, but not an issue really

Reply:
Let's see how it goes and adjust if we observe problems. FWIW I also tried using `--depth` but for some reason that kept timing out...